CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Restify: Fix for calling next() ([@jadkap](https://github.com/jadkap)) on [#285](https://github.com/apollostack/graphql-server/pull/285)
 * Update GraphiQL to version 0.9.1 ([@ephemer](https://github.com/ephemer)) on [#293](https://github.com/apollostack/graphql-server/pull/293)
 * Add AWS Lambda Integration [#101](https://github.com/apollostack/graphql-server/issues/101)
+* Add Zeit Micro Integration [#324](https://github.com/apollographql/graphql-server/issues/324)
```

Comment:
Have we really not released since adding AWS lamdba or did we (I?) just forget to update the changelog with the latest version?